be/test/olap/skiplist_test.cpp

```diff
@@ -268,7 +271,7 @@ class ConcurrentTest {
     ConcurrentTest()
             : _mem_tracker(new MemTracker(-1)),
               _mem_pool(new MemPool(_mem_tracker.get())),
-              _list(TestComparator(), _mem_pool.get(), false) {}
+              _list(new TestComparator(), _mem_pool.get(), false) {}
```

Comment:
Add `~ConcurrentTest()` to delete `_list`

Reply:
_list is an object, is it needs to delete it?

Reply:
Yes, you pass a raw pointer to the skiplist, and nowhere to delete this `TestComparator`

Reply:
done